docs/sources/calendar-schedules/web-schedule/calendar-export/index.md

```diff
@@ -0,0 +1,65 @@
+canonical: https://grafana.com/docs/oncall/latest/calendar-schedules/web-schedule/calendar-export/
+description: "Learn how to export an on-call schedule from Grafana OnCall"
+keywords:
+  - Grafana
+  - oncall
+  - on-call
+  - calendar
+  - iCal export 
+weight: 500 
+---
+
+# Export on-call schedules
+
+Export on-call schedules from Grafana OnCall to your preferred calendar app with a one-time secret iCal URL. The schedule export allows you to add on-call schedules to your existing calendar to view on-call shifts alongside the rest of your schedule.  
+
+There are two schedule export options available:
+
+- **On-call schedule export** - Exports all on-call shifts for a particular schedule, including rotations, overrides, and assigned users.
+- **User-specific schedule export** - Exports assigned on-call shifts for a particular user. Use this export option to add your assigned on-call shifts to your calendar. 
+
+## Export an on-call schedule
+
+Use this export option to add all on-call shifts associated with a schedule to a calendar. Best for a team or shared calendars. 
+
+To export a schedule from Grafana OnCall:
+
+1. In Grafana OnCall, navigate to the **Schedules** tab.
+1. Open the schedule you’d like to export by clicking on the schedule name.
+1. Click **Export** in the upper right corner, then click **+ Create iCal link** to generate a secret iCal URL.
+1. Copy the iCal link and store it somewhere you’ll remember. Once you close the schedule export window, you won't be able to access the iCal link. 
+1. Open your calendar settings to add a calendar from a URL (This step varies based on your calendar app).
+
+## Export a user on-call schedule
+Use this export option to add your assigned on-call shifts to your calendar. Best for personal calendars.
+
+To export your on-call schedule:
+
+1. In Grafana OnCall, navigate to the **Users** tab.
+1. Click **View my profile** in the upper right corner.
+1. From the **User Info** tab, navigate to the iCal link section.
+1. Click **+ Create iCal link** to generate your secret iCal URL.
+1. Copy the iCal link and store it somewhere you’ll remember. Once you close your user profile, you won't be able to access the iCal link again.
+1. Open your calendar settings to add a calendar from a URL (This step varies based on your calendar app).
+
+## Revoke an iCal export link 
+
+iCal links are displayed upon creation, and users are advised to copy their link and store it for future reference. To ensure the security of your and your teams' calendar data, after an iCal link is generated, the link is hidden and cannot be accessed again. 
+
+If you need to revoke an iCal link, you can do so anytime. By doing so, any calendar that references the revoked link will lose access to the calendar data.
+
+>**Note**: Use caution when revoking an iCal link associated with shared on-call schedules. If you aren't the creator of the existing iCal link, check with your teammates before revoking it. 
```

Comment:
Ok gotcha, I expected schedule level exports to be org-wide. 

Quick clarifying questions: Say User 1 generates an export link from a schedule itself rather than from their user profile - Then User 2 goes to that same schedule to generate an export link, is User 2 is able to generate another link to the same schedule?

Reply:
Yes, each user will be able to generate (and revoke) their own export link for the same schedule (links will be different).